cmd/junod/main.go

```diff
@@ -20,7 +21,7 @@ func main() {
 		// this line is used by starport scaffolding # root/arguments
 		cmdOptions...,
 	)
-
+	rootCmd.AddCommand(tmcmds.RollbackStateCmd)
```

Comment:
you can add to cmdOptions before `NewRootCmd`
```suggestion
	cmdOptions = append(cmdOptions, cosmoscmd.AddSubCmd(tmcmds.RollbackStateCmd))
```

Reply:
Done in 5b377c7